app/uk/gov/hmrc/ctcguaranteebalancerouter/connectors/EISConnector.scala

```diff
@@ -133,7 +134,7 @@ class EISConnectorImpl(
             .map[Either[ConnectorError, A]] {
               response: HttpResponse =>
                 response.status match {
-                  case INTERNAL_SERVER_ERROR =>
+                  case FORBIDDEN =>
```

Comment:
I am bit confused and not sure if I am correct. So if it's under this case condition then do we not need to change the ConnectorError.UnexpectedError with ConnectorError.ForbiddenError? Just like we did in transit-movements-router? This is just my observation and have asked in refinement if you remember is it done for router then Dan said yes. So are we not expecting same behaviour here as well? Just like LRN story one you have done. See below code from transit-movemetns-router
```
              (error.statusCode, error.message) match {
                case (ErrorCode.Forbidden.statusCode | ErrorCode.InternalServerError.statusCode, lrnRegexPattern(lrn, _)) =>
                  Left(
                    RoutingError.DuplicateLRNError(s"LRN $lrn has previously been used and cannot be reused", ErrorCode.Conflict, LocalReferenceNumber(lrn))
                  ) ```

Reply:
Both falls under bit different scenario as we have already have FORBIDDEN status and in addition to that only for that LRN scenario can expect in 500 status code as well, hence included for both the status. Here in the GB the existing response code is 500 and now it is changed from INTERNAL_SERVER_ERROR to FORBIDDEN, hence this is the expected change & still we are not changing the response error code to the end-user so its the same and I haven't changed that one. Moreover I don't see that changing that in the ticket description as well. Hope this clarifies you.

Reply:
I think that's fine. Will see if anyone complain about anything.Thanks